development/scripts/clone-repos.sh

```diff
@@ -21,7 +20,5 @@ do
     cd "${repo}"/
     git remote add upstream git@github.com:opensecuritycontroller/"${repo}".git
     git remote set-url --push upstream no_push
-    cd ../"${repo}"/.git/hooks 
-    ln -s ../../hooks/pre-push .
-    cd ../../../
+    cd ../

```

Comment:
I see,  I personally like to ensure the master of my fork is always a mirror or subset of the origin master. I find it useful to have a clean baseline branch. For any changes I usually just create a short lived branch.  But that might be too much enforcement? If  it is common to leave that open to the owner of the fork you can remove it.